ThreeDSecure/src/test/java/com/braintreepayments/api/threedsecure/MockCardinalClientBuilder.java

```diff
@@ -13,6 +13,7 @@
 
 import org.mockito.stubbing.Answer;
 
+@Deprecated
```

Comment:
General question, are there other tests still using this one or can it be removed now that `ThreeDSecureClientUnitTest` has been converted?